src/pages/DrinkListPage.js

```diff
@@ -12,45 +14,56 @@ export default function DrinkListPage({
   currentFilter,
 }) {
   return (
-    <DrinklistContainer>
-      <Searchbar
-        handleChangeSearch={handleChangeSearch}
-        searchValue={searchValue}
-      />
-      <FilterDrinks
-        handleChangeFilter={handleChangeFilter}
-        currentFilter={currentFilter}
-      />
-      {drinks &&
-        drinks
-          .filter(drink =>
-            drink.strDrink.trim().toLowerCase().includes(searchValue.trim())
-          )
-          .filter(
-            drink =>
-              drink.strAlcoholic.toLowerCase() === currentFilter ||
-              currentFilter === 'all'
-          )
-          .map(drink => (
-            <DrinkCard
-              key={drink.idDrink}
-              drink={drink}
-              toggleBookmark={toggleBookmark}
-            />
-          ))}
-      {drinks &&
-        drinks.filter(drink =>
-          drink.strDrink.trim().toLowerCase().includes(searchValue)
-        ).length < 1 && (
-          <EmptyState>There are no results for your search input...</EmptyState>
-        )}
-    </DrinklistContainer>
+    <PageContainer>
+      <Header>Cocktail Night</Header>
+      <DrinklistContainer>
+        <Searchbar
+          handleChangeSearch={handleChangeSearch}
+          searchValue={searchValue}
+        />
+        <FilterDrinks
+          handleChangeFilter={handleChangeFilter}
+          currentFilter={currentFilter}
+        />
+        {drinks &&
+          drinks
+            .filter(drink =>
+              drink.strDrink.trim().toLowerCase().includes(searchValue.trim())
```

Comment:
I think it would be more readable here as well to declare a const above with the filtering stuff you are doing here and then filter / map over these variables here and below.

Reply:
here refactoring user story too